data-node/src/test/java/org/graylog/datanode/filesystem/index/IndicesDirectoryParserTest.java

```diff
@@ -46,15 +48,15 @@ void testOpensearch2() throws URISyntaxException {
         Assertions.assertThat(result.nodes())
                 .hasSize(1)
                 .allSatisfy(node -> {
-                    Assertions.assertThat(node.nodeVersion()).isEqualTo("2.10.0");
+                    Assertions.assertThat(node.nodeVersion()).isEqualTo(OPENSEARCH_VERSION_IN_DATANODE);
                     Assertions.assertThat(node.indices())
                             .hasSize(6)
                             .extracting(IndexInformation::indexName)
                             .contains(".opensearch-sap-log-types-config", ".plugins-ml-config", "graylog_0", ".opensearch-observability", ".opendistro_security", "security-auditlog-2023.11.24");
 
                     final IndexInformation graylog_0 = node.indices().stream().filter(i -> i.indexName().equals("graylog_0")).findFirst().orElseThrow(() -> new RuntimeException("Failed to detect graylog_0 index"));
 
-                    Assertions.assertThat(graylog_0.indexVersionCreated()).isEqualTo("2.10.0");
+                    Assertions.assertThat(graylog_0.indexVersionCreated()).isEqualTo(OPENSEARCH_VERSION_IN_DATANODE);
```

Comment:
The index directory has been created by 2.10 and will stay in that version if we won't regenerate the files with a newer one. This is why the test is now failing here.

Reply:
reverted, thx.